gbpservice/neutron/services/servicechain/drivers/oneconvergence_appliance_driver.py

```diff
@@ -270,6 +273,44 @@ def _fetch_template_and_params(self, context, sc_instance,
                 stack_template[resources_key]['Firewall'][properties_key][
                     'description'] = str(firewall_desc)
             else:
+                #For remote vpn - we need to create a implicit l3 policy
```

Comment:
Can we modularize this logic in an separate method,  instead of inline it here.
mainline code path will be more readable:
something along the lines: create_rvpn_l3_policy()